### ISSUE TITLE
should we provide an config to show line number  in log? 

### ISSUE THREAD

**Issue by leizhiyuan** [alipay/sofa-mosn#399]
### The New Feature

log in golang can set Lshortfile to show current number, it is expensive, but it can help us identify code quickly. 

now we always search the log content to find the code, or can find many codes which will print the specify log content.

maybe we can open it in local machine or dev environment?  

```
log.SetFlags(log.LstdFlags | log.Lshortfile)
```

it will call
```
_, file, line, ok = runtime.Caller(calldepth)
```

then log will like

```
// "[date] [time] boltcodec.go:11: Example"

```
### Your scenes

describe your use scenes (why need this feature)

### Your advice

describe the advice or solution of this new feature

### Environment
+ SOFAMosn Version

**Comment by nejisama** [comment]
fix in #246

**Comment by leizhiyuan** [comment]
it is different with this issue。

**Comment by nejisama** [comment]
fix in #246 by the way
